### ISSUE TITLE
Install sphinx to build html docs (it is installed)

### ISSUE THREAD

**Issue by SkiftCreative** [paver/paver#168]
Following the docs, I am trying to get paver setup to run sphinx to setup the docs. Here is my output

```
> paver html
---> paver.doctools.html


Captured Task Output:
---------------------

---> paver.doctools.html

Build failed running paver.doctools.html: install sphinx to build html docs
```

So I cannot figure out why I get the error. As I do have sphinx installed. I have this in my `pavement.py`:

```
"""
Setup script.
"""

import sys

import os
from distutils.core import Command
from setuptools import setup, __version__
from setuptools.command.test import test as TestCommand

from paver.easy import *
import paver.doctools
from paver.setuputils import setup

options(
    sphinx=Bunch(
        apidir='api',
        builddir="build",
        docroot='docs',
        sourcedir='source'
    )
)


IS_PROD = os.getenv('GAE_VM', False)

install_requires = [
    'paver>=1.2.0,<3.0.0'
]

dev_install_requires = [
    'alabaster==0.7.7',
    'Babel==2.3.4',
    'codecov==2.0.3',
    'coverage==4.0.3',
    'docutils==0.12',
    'imagesize==0.7.1',
    'Jinja2==2.8',
    'MarkupSafe==0.23',
    'pluggy==0.3.1',
    'py==1.4.31',
    'Pygments==2.1.3',
    'pytz==2016.4',
    'requests==2.10.0',
    'six==1.10.0',
    'snowballstemmer==1.2.1',
    'Sphinx==1.4.1',
    'tox==2.3.1',
    'virtualenv==15.0.1',
]

if not IS_PROD:
    install_requires += dev_install_requires

setup(
    author='Shawn McElroy',
    author_email='shawn@skift.io',
    description='newapp',
    download_url='',
    install_requires=install_requires,
    license='Apache License (2.0)',
    name='newapp',
    packages=[
        'founder',
    ],
    scripts=[],
    tests_require=[
        'codecov>=2.0.3,<3.0.0',
        'coverage>=4.0.3,<5.0.0',
        'Sphinx>=1.4.1,<2.0.0',
        'tox>=2.3.1,<3.0.0',
        'virtualenv>=15.0.1,<16.0.0'
    ],
    url='',
    version='0.1.0'
)

```

i have paver and sphinx installed:

```
> pip list
alabaster (0.7.7)
Babel (2.3.4)
codecov (2.0.3)
coverage (4.0.3)
docutils (0.12)
newapp (0.1.0, /Users/shawn/code/src/gitlab.com/secureaf/newapp)
imagesize (0.7.1)
Jinja2 (2.8)
MarkupSafe (0.23)
Paver (1.2.4)
pip (8.1.2)
pluggy (0.3.1)
py (1.4.31)
Pygments (2.1.3)
pytz (2016.4)
requests (2.10.0)
setuptools (24.0.2)
six (1.10.0)
snowballstemmer (1.2.1)
Sphinx (1.4.1)
tox (2.3.1)
virtualenv (15.0.1)
wheel (0.29.0)
```

**Comment by SkiftCreative** [comment]
Closing this as it was due to an issue I had with my environment.

**Comment by Almad** [comment]
@SkiftCreative would you mind sharing which ones for accidental googlers? :)

**Comment by SkiftCreative** [comment]
Yea. My paver was installed on the global python using `/usr/local/bin/pip` (python2). So when I ran it, paver was looking at the global python2 site-packages for `sphinx` this would happen the same if it was intalled with the python3 version at `/usr/local/bin/pip3`. The reason this happened, is because that PATH was before my virtual environment. So once I ran `/usr/local/bin/pip uninstall paver` and ran `paver html` again, it properly saw the version of paver in my virtual environment, and ran correctly.
